netutils/config/parser.py

```diff
@@ -978,7 +979,7 @@ def _is_section_title(self, line):  # pylint: disable=no-self-use
             return True
         return False
 
-    def _get_section_title(self, line):  # pylint: disable=no-self-use
+    def _get_section_title(self, line: str) -> t.Union[str, bool]:  # pylint: disable=no-self-use
```

Comment:
In the `False` case, should we return an empty string? This would be a somewhat breaking change, but the empty string is still fals-ey (`bool("") == False`).